.mergify.yml

```diff
@@ -41,8 +41,10 @@ pull_request_rules:
         - author≠@dont-squash-my-commits
         - or:
           # only require docs checks if docs files changed
-          - status-success=docs-build
           - -files~=^docs/
+          - and:
+            - status-success=build & deploy
+            - check-skipped=error_reporting
```

Comment:
Do you know if there is a way to specify the workflow of the github-action job? I didn't see one.
If not, we might need to make job-names more specific, as I see we're adding another `error_reporting` in #29009

Reply:
I think the job name is the only thing they can recognize.

btw, is it okay to remove `- check-skipped=error_reporting` 🤔 
the error_reporting only works when any previous steps failed and I think we can ignore its status.

Reply:
I *think* that we can adjust to only one criterion: `- status-success=build & deploy`
Any build that touches docs files must run `build & deploy`, right?

Also, how about we rename that job to "build & deploy docs" to be more explicit for this use?

Reply:
Okay, updated to the single criterion @yihau . (Needs rebase on #29072 )

Reply:
just merged 😈